cmd/agent/api/internal/agent/agent.go

```diff
@@ -256,6 +258,72 @@ func streamLogs(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func streamEventPlatform(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is nearly identical to streamLogs, except for different value of the receiver. Would it make sense to refactor the code to avoid this duplication?